modules/rostests/apitests/win32u/win32u_2k3sp2/CMakeLists.txt

```diff
@@ -7,5 +7,5 @@ add_library(win32u_2k3sp2 MODULE
     ${win32u_2k3sp2_asm}
     ${CMAKE_CURRENT_BINARY_DIR}/win32u_2k3sp2.def)
 
-set_module_type(win32u_2k3sp2 module)
+set_module_type(win32u_2k3sp2 win32dll IMAGEBASE default)
```

Comment:
This reverts 45ec49c. Is that alright? Cc @katahiromz

Reply:
Questioned part reverted.

Reply:
Yes, that was a mistake.